requirements-dev.txt

```diff
@@ -31,3 +31,4 @@ pytest-cov>=2.6.1
 coveralls>=1.3
 freezegun>=0.3.12
 moto>=1.3.7
+sphinx-rtd-theme>=0.4.3
```

Comment:
Move to docs/requirements.txt

Reply:
Removed - this requirement already appears in docs/requirements.txt